src/client/app/actions/admin.ts

```diff
@@ -98,6 +96,9 @@ function fetchPreferences(): Thunk {
 				}
 				if (preferences.defaultLanguage !== state.options.selectedLanguage) {
```

Comment:
The logic seems to work fine but I think having a comment(s) about what the cases are and what/why the value specified is set would be good.